epub33/rs/index.html

```diff
@@ -1021,29 +1021,23 @@ <h4>The <code>rendition:page-spread-*</code> Properties</h4>
 					<p>The <span id="page-spread-left"><code>rendition:page-spread-left</code></span> property indicates
 						that the given spine item SHOULD be rendered in the left-hand slot in the spread, and <span
 							id="page-spread-right"><code>rendition:page-spread-right</code></span> that it SHOULD be
-						rendered in the right-hand slot. The <span id="page-spread-center"
-								><code>rendition:page-spread-center</code></span> property indicates that the synthetic
-						spread mode SHOULD be overridden and a single viewport rendered and positioned at the center of
-						the screen.</p>
+						rendered in the right-hand slot.</p>
+
+					<p>Reading Systems MUST handle the <span id="page-spread-center"
+								><code>rendition:page-spread-center</code></span> property as though the EPUB Creator
+						has specified the <a href="#def-spread-none"><code>spread-none</code> property</a> on the
+							<code>itemref</code> element.</p>
```

Comment:
Note that if `page-spread-center` is labeled as "deprecated" then the _MUST_ statement is to be made a bit more precise. Per the [definition of deprecated](https://w3c.github.io/epub-specs/epub33/core/#app-overview-unsupported) an RS _MAY_ implement it but not more. Ie, this statement should say something like "if the RS implements `page-spread-center` then it _MUST_…"

Reply:
I'm not sure that works for aliasing, as it seems to suggest the reading system ignore it if it only supports spread-none even though it's an alias. Or that it can support the property independent of it being an alias (i.e., you support spread-none if you support rendition:page-spread-center, since only the former really exists).

Maybe "handle" is the wrong word here, as my intention was only to recognize it as the other property and do what that property requires (if supported).

What about reversing the relationship and dropping the handling, which flows from recognizing the value as an alias, so it becomes:

"Reading Systems that support the `spread-none` property MUST recognize the `rendition:page-spread-center` property as an alias for it, otherwise they MUST ignore the `rendition:page-spread-center` property."

Reply:
Yep, @mattgarrish, the reformulation works for me, but... don't we have to rework the content document as well? Namely, is the statement in  [§5.2.4](https://www.w3.org/TR/epub-33/#page-spread) on `rendition:page-spread-center` still correct? What is the intention there? Shouldn't it mean the same as the definition of `spread-none` in the previous section?

Cc @wareid

Reply:
> Shouldn't it mean the same as the definition of spread-none in the previous section?

It should. With this update it now reads:

> rendition:page-spread-center [DEPRECATED]
> Use of the rendition:page-spread-center property is deprecated. EPUB Creators should use the spread-none property to disable spread behavior in Reading Systems.

I followed the pattern we used for rendition:spread portrait here. (But the deprecated label needs reformatting looking at how we applied it elsewhere.)

Reply:
Shoot. I should have realized that you also edited the content document... I will be silent now!